learning_a_programming_language/loop_operator/m.c

```diff
@@ -0,0 +1,18 @@
+#include <stdio.h>
+
+int main(){
+	int n = 0;
+	int b = 0;
+	int c = 0;
+	scanf("%d", &n);
```

Comment:
add empty line after definitions
```suggestion
	
	scanf("%d", &n);
```